src/app/containers/Ad/Amp/index.jsx

```diff
@@ -135,13 +98,25 @@ const AmpAd = () => {
             >
               {label}
             </StyledLink>
-            <amp-ad {...ampAdPropsMobile({ service })} />
-            <amp-ad {...ampAdPropsDesktop({ service })} />
+            <AdSlot
+              viewportType="mobile"
+              service={service}
+              slotType={slotType}
+            />
+            <AdSlot
```

Comment:
We have an object for mpu mobile:
```
  mpu: {
    mobile: {
      width: '300',
      height: '250',
      adMultiSize: '300x250',
    },
  },
```

However when we pass in viewportType `desktop`, with slotType `mpu` I believe this will be undefined

Reply:
Further refactors done so that AdSlot handles the viewport and abstracts that away. Fixes a reusability issue and is more in line with GNL.